auctions/dutch-auction/README.md

```diff
@@ -0,0 +1,5 @@
+This is an implementation of a Dutch Auction in Sway. A dutch auction is one where price falls gradually from an initially high price, and the first person to bid wins the auction.
```

Comment:
These are just a few lines without any structure. Add some headings or more information etc. A block of text is not going to catch the eye enough for someone to read it